prac_06/used_cars.py

```diff
@@ -0,0 +1,26 @@
+"""CP1404/CP5632 Practical - Client code to use the Car class."""
+# Note that the import has a folder (module) in it.
+
+from prac_06.car import Car
+
+
+def main():
+    """Demo test code to show how to use car class."""
+    my_car = Car(180)
+    my_car.drive(30)
+    print("fuel =", my_car.fuel)
+    print("odo =", my_car.odometer)
+    print(my_car)
+
+    print("Car {}, {}".format(my_car.fuel, my_car.odometer))
+    print("Car {self.fuel}, {self.odometer}".format(self=my_car))
```

Comment:
maybe use a different key then 'self', it did throw me off for a second how you were using 'self.' outside of the class.

Reply:
You're dead on, that's terrible! Thanks for spotting that, I'll sort it out and merge.